### DISCUSSION TITLE
Object viewer should show low level methods at the top

### DISCUSSION THREAD

**Discussion by gshotwell** [posit-dev/positron#4332] [Feedback]
This is a bit hard to describe, and I'm not sure how I'd accomplish it but here goes. 

When viewing objects in the object viewer Positron very helpfully displays all of the methods attached to that object. The issue is that when an object inherits from something like a pandas Series, all of the methods are mixed together. It would be better if direct and inherited methods were differentiated in the tree structure. 

For example consider in this class I want `my_method` to be at the top with all of the pandas methods hidden in a drop down. I could also see a similar UI being really handy for S4 classes because it would maybe be easier to keep track of multiple inheritance. 

```
import pandas as pd

class CustomSeries(pd.Series):
    @property
    def _constructor(self):
        return CustomSeries
    
    def my_method(self):
        return self ** 2

# Example usage
data = [1, 2, 3, 4, 5]
custom_series = CustomSeries(data)
```

**Comment by seeM** [comment]
Thanks for the feedback @gshotwell! I've filed this as an enhancement request: https://github.com/posit-dev/positron/issues/4351.
